docs/team/fsgmhoward.md

```diff
@@ -0,0 +1,81 @@
+# ZH Liu — Project Portfolio Page
+
+## Overview
+Patient Manager is a _Command Line Interface_ (CLI) application for **_general practitioners_** (GP)
+who work in polyclinics to manage their patient list.
+
+My teammate, together with me, developed this application as a team project for CS2113T module of NUS.
+This project aimed to develop a program targetting a current problem using Java with Object-oriented
```

Comment:
Typo here, should be "targeting"

Reply:
Thank you! This has been resolved in the latest commit. My local proof reader didn't report it as a typo. It probably understood it as British spelling. After checking articles on this, I guess you are correct.